frontend/views/containers/proposals/ChangeVotingRules.vue

```diff
@@ -87,10 +87,14 @@ export default ({
       'currentGroupId'
     ]),
     ...mapGetters([
+      'ourUsername',
       'groupShouldPropose',
       'groupProposalSettings',
       'groupSettings'
     ]),
+    disabled () {
+      return !this.groupShouldPropose && this.ourUsername !== this.groupSettings.groupCreator
+    },
```

Comment:
Why was this added?

Reply:
So we don't allow a group member to update voting rules when group has less than 3 members and he is not group creator.

Reply:
Users can open the modal even though we disable the menu item.
They can copy the URL (e.g, http://localhost:8000/app/dashboard?rule=percentage&modal=ChangeVotingRules) and open it directly in the browser. So I thought we should check the permission inside the modal and should do one of the following two things.
- Close the modal
- Disable the submission